tests/Feature/PrepareStreamsTest.php

```diff
@@ -63,6 +63,8 @@ public function it_orders_streams_from_current_to_upcoming(): void
     /** @test */
     public function it_orders_past_streams_from_latest_to_oldest(): void
     {
```

Comment:
Thanks, I think we can remove this test now. We already check the ordering in the tests for the home or archive page. Please remove it.